api/src/exceptions/unexpected-response.ts

```diff
@@ -0,0 +1,7 @@
+import { BaseException } from '@directus/shared/exceptions';
+
+export class UnexpectedResponseException extends BaseException {
+	constructor(message: string) {
+		super(message, 500, 'UNEXPECTED_RESPONSE');
```

Comment:
Should we do a 503 here, as it's the external service that's unavailable?

Reply:
It's not unavailable though. The issue is that it is available when it shouldn't be. We expect a failure and never get it.

Reply:
That being said, I don't mind what we use

Reply:
500 is just a generic "internal" server error, but this isn't an unknown error that's happening within Directus itself, but rather an unexpected error in the related service. I agree that a 503 is technically not 100% accurate, but it feels closer to this particular situation than a 500 to me 🤔

Reply:
Sure, we can go for that then. Nobody outside of MiniDigger is ever gonna see this anyway. 😂

Reply:
I feel attacked! 
Sadly I don't think we are the only IBM zOS customer thats left for them to milk...